src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -280,7 +280,7 @@ public static unsafe void OutputEncodingPreamble()
         }
     }
 
-    [Fact]
+    [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
     public static unsafe void OutputEncoding()
```

Comment:
Is `PlatformDetection.IsThreadingSupported` considered just as some common approach to disable all the failing tests at the moment regardless the area?

Reply:
It's used for all tests that rely on starting new threads since WebAssembly doesn't support more than one thread. We have an exception with a message like "Cannot start threads on this runtime" when someone tries to start a new thread .

Reply:
This particular Console test fails with PNSE. If we use conditional fact then I need to close https://github.com/dotnet/runtime/pull/37645

Reply:
I don't know which approach is preferred, but I can remove this change if #37645 is preferred. This PR was based solely off the failing test list, without looking into the test failures, so it could be that other tests fail with PNSE as well.

Reply:
Hmm we should only disable tests in this PR that failed with the `"Cannot start threads on this runtime"`, not any PNSE, since we use PNSE for other cases that have nothing to do with threading. Thanks @MaximLipnin for noticing it.